lib/presentation/widget/two_factor_authen/two_factor_widget.dart

```diff
@@ -0,0 +1,123 @@
+  @override
+  _TwoFactorAuthenticationWidgetState createState() => _TwoFactorAuthenticationWidgetState();
+}
+
+class _TwoFactorAuthenticationWidgetState extends State<TwoFactorAuthenticationWidget> {
+  final imagePath = getIt<AppImagePaths>();
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      resizeToAvoidBottomInset: false,
+      resizeToAvoidBottomPadding: false,
+      backgroundColor: AppColor.primaryColor,
+      body: GestureDetector(
+        onTap: () => FocusScope.of(context).unfocus(),
+        child: SafeArea(
+          child: Stack(children: [
+            SingleChildScrollView(
+              reverse: false,
+              child: Padding(
+                padding: EdgeInsets.only(bottom: MediaQuery.of(context).viewInsets.bottom),
+                child: Container(
+                  alignment: Alignment.topCenter,
+                  padding: EdgeInsets.only(top: 100),
+                  child: Column(
+                    children: [
+                      LinShareSloganBuilder()
+                          .setSloganText(AppLocalizations.of(context).login_text_slogan)
+                          .setSloganTextAlign(TextAlign.center)
+                          .setSloganTextStyle(TextStyle(color: Colors.white, fontSize: 16))
+                          .setLogo(imagePath.icLoginLogo)
+                          .build(),
+                      SizedBox(height: 80),
+                      PinCodeWidget.twoFactorAuthen(
+                        title: AppLocalizations.of(context).second_factor_authentication,
+                        titleStyle: _titleTextStyle,
+                        subTitle: AppLocalizations.of(context).input_6_digit_from_free_otp,
+                        subTitleStyle: _subTitleTextStyle,
+                        pinCodeFieldBackgroundColor: AppColor.primaryColor,
+                        pinBackgroundColor: AppColor.pinCodePadBackgroundColor,
+                        pinTextStyle: _padTextStyle,
+                        errorText: AppLocalizations.of(context).please_fill_up_all_numbers,
+                        errorStyle: _errorTextStyle,
+                        buttonText: AppLocalizations.of(context).submit,
+                        buttonBackgroundColor: AppColor.pinCodeSubmitButtonBGColor,
+                        buttonStyle: _buttonActionTextStyle,
+                        onButtonClick: _sendOtpToVerify
+                      )
+                    ],
+                  ),
+                ),
+              ),
+            ),
+            ToolbarBuilder(
+                Key('two_factor_arrow_back_button'),
+                contentPadding: EdgeInsets.only(left: 10),
+                actionIcon: imagePath.icArrowBack,
+                onButtonActionClick: _handleBackPress)
+                .build(),
+          ]),
+        ),
+      ),
+    );
+  }
+
+  // TODO: invoke popBack in view model
+  void _handleBackPress() {
+    print('back press...');
```

Comment:
remove it

Reply:
Sr, my bad. This file is redundant, i'll remove it in fixup commit